### PR TITLE
feature(ui-ux): defi testing v2

### PR DESCRIPTION
**Pull request by G9000**
#### What kind of PR is this?:
/kind feature

#### What this PR does / why we need it:
Update DFI testing to v2

#### Which issue(s) does this PR fixes?:

Fixes #

#### Developer Checklist:

- [x] Read your code changes at least once
- [x] Tested on iOS/Android device (e.g, No crashes, library supported etc.)
- [x] No console errors on web
- [x] Tested on Light mode and Dark mode*
- [x] Your UI implementation visually matched the rendered design*
- [ ] Unit tests*
- [ ] Added e2e tests*
- [ ] Added translations*

### CODE REVIEW

File: mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundWallet.tsx [hunk 1]
```diff
@@ -18,54 +21,119 @@ export function PlaygroundWallet (): JSX.Element | null {
...
+        dark={tailwind('bg-mono-dark-v2-00')}
+        light={tailwind('bg-mono-light-v2-00')}
+        style={tailwind('rounded-lg-v2 px-5')}
+      >
+        {
+        dataLists.map((dataList, index) => (
+          <PlaygroundAction
+            key={index}
+            // eslint-disable-next-line react/jsx-handler-names
+            onPress={dataList.onPress} // let it be just for dfi testing. it still working
```
Comment (pierregee) [inline mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundWallet.tsx:127]: can remove this comment? 🤔 and also what does it mean?

File: mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundConnection.tsx [hunk 2]
```diff
@@ -1,60 +1,22 @@
...
-            {' '}
-
-            {network}
-          </ThemedText>
-        </Text>
-
-        <ThemedText>
-          Blocks:
-          {' '}
-          {count === 0 ? '...' : <Text testID='current_block_count'>{count}</Text>}
```
Comment (pierregee) [inline mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundConnection.tsx:45]: image
Failing test due to this

File: mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundOperations.tsx [hunk 3]
```diff
@@ -34,41 +37,80 @@ export function PlaygroundOperations (): JSX.Element {
...
+      testID: 'playground_wallet_fetch_balances'
+    },
+    {
+      title: 'Generate block',
+      onPress: async (): Promise<void> => {
+        await rpc.call('generatetoaddress', [10, 'mswsMVsyGMj1FzDMbbxw2QW3KvQAv2FKiy'], 'number')
+      },
+      rhsChildren: (): JSX.Element => {
+        return (
+          <ThemedIcon
```
Comment (pierregee) [inline mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundOperations.tsx:70]: It should be + icon here based on design

Comment (pierregee) [reply mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundOperations.tsx:70]: Same with the other top up tokens 
image

**Review by pierregee — CHANGES_REQUESTED** [review]

File: mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundConnection.tsx [hunk 4]
```diff
@@ -1,60 +1,22 @@
...
   return (
-    <View>
-      <PlaygroundTitle
-        status={{ online: connected, offline: !connected }}
-        title='Connection'
-      />
+    <ThemedViewV2 style={tailwind('px-5 mt-8 mb-6')}>
+      {
+        isPlayground(network) && (
+          <ThemedTextV2 style={tailwind('text-sm')}>
```
[the remark this answers is not in the document]
Comment (G9000) [reply mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundConnection.tsx:14]: done

File: mobile-app/app/screens/PlaygroundNavigator/components/PlaygroundAction.tsx [hunk 5]
```diff
@@ -1,32 +1,22 @@
...
+import { ThemedTouchableListItem, ThemedTextV2 } from '@components/themed'
+import { StyleProp, ViewStyle } from 'react-native'
 interface PlaygroundActionProps {
-  testID: string
   title: string
-  onPress: () => void
+  rhsChildren?: () => ReactElement | ReactComponentElement<any>
+  isLast: boolean
+  testID?: string
+  containerStyle?: StyleProp<ViewStyle>
```
[the remark this answers is not in the document]
Comment (G9000) [reply mobile-app/app/screens/PlaygroundNavigator/components/PlaygroundAction.tsx:9]: done

File: mobile-app/app/screens/PlaygroundNavigator/components/PlaygroundAction.tsx [hunk 6]
```diff
@@ -1,32 +1,22 @@
...
-      <View style={tailwind('px-4')} />
-
-      <ThemedIcon
-        iconType='MaterialIcons'
-        name='chevron-right'
-        size={24}
-      />
-    </ThemedTouchableOpacity>
+    <ThemedTouchableListItem isLast={isLast} testID={testID} onPress={onPress}>
+      <ThemedTextV2 style={containerStyle}>
```
[the remark this answers is not in the document]
Comment (G9000) [reply mobile-app/app/screens/PlaygroundNavigator/components/PlaygroundAction.tsx:16]: done

File: mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundWallet.tsx (hunk 1 above)
Comment (G9000) [reply mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundWallet.tsx:127]: done cleanup

File: mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundConnection.tsx (hunk 2 above)
Comment (G9000) [reply mobile-app/app/screens/PlaygroundNavigator/sections/PlaygroundConnection.tsx:45]: done

File: mobile-app/app/screens/PlaygroundNavigator/components/PlaygroundAction.tsx [hunk 7]
```diff
@@ -1,32 +1,23 @@
...
+import { ReactElement, ReactComponentElement } from 'react'
+import { ThemedTouchableListItem, ThemedTextV2 } from '@components/themed'
+import { StyleProp, TextStyle } from 'react-native'
 import { tailwind } from '@tailwind'
-
 interface PlaygroundActionProps {
-  testID: string
   title: string
-  onPress: () => void
+  rhsChildren?: () => ReactElement | ReactComponentElement<any>
```
Comment (pierregee) [inline mobile-app/app/screens/PlaygroundNavigator/components/PlaygroundAction.tsx:7]:
```suggestion
  rhsChildren?: () => JSX.Element
```

Comment (G9000) [reply mobile-app/app/screens/PlaygroundNavigator/components/PlaygroundAction.tsx]: done

**Review by pierregee — APPROVED** [review]

**Pull request merged**